### DISCUSSION TITLE
关于php文件权限提示信息的建议(wordpress无法上传文件)

### DISCUSSION THREAD

**Discussion by defeatedperson** [1Panel-dev/1Panel#6154] [Ideas]
### 1Panel 版本

1.9.3

### 请描述您的需求或者改进建议

在php相关的页面都加上
【php站点故障？戳此处查看教程】提示
我所在的技术群里面一堆遇到这种情况的小白

目前的提示不是很明显，内容也不是很易懂(对于新用户)

### 请描述你建议的实现方案

可以考虑在站点管理页面，
就是首页点击【网站】那个页面

把php目录权限的提示移到这里，
而不是站点详细设置页面(目录设置-保存)

提示文案改成:
xx站点，php设置疑似有问题，可能导致无法上传文件等问题，点击此处查看解决方法

点击这个提示就能跳转1panel文档的图文教程
(文档库里面可以加上这个教程)

最后，1panel很不错，但还是有一点点不适合新手，毕竟bt不需要这样操作，祝1panel越来越好!

### 附加信息

附加提示:
同理，也可以给数据库加上【无法连接数据库？戳此处查看教程】的提示

我提的建议主要针对小白用户，多数懂点技术的大佬应该都没这些问题
但，个人觉得，降低小白新用户门槛也是应该要做的

**Comment by ltzXiaoYanMo** [comment]
我不太认为这是一个很好的想法...

~~首先你搜索绝对能得出答案~~

这并非假话，因Linux系统的开源性使你大多数的问题都能解决。

1Panel官方文档并非能解决所有问题，具体你得无限的琢磨，这就是我们开发者的探索。

**Comment by defeatedperson** [comment]
> 我不太认为这是一个很好的想法...

个人的建议是讲已经有的提示，从详细设置，改到站点管理页面，让用户可以直接看到问题所在

新用户往往不知道需要点站点设置-目录的保存按钮，
这会导致例如Wordpress之类的php项目无法上传文件

**Comment by ltzXiaoYanMo** [comment]
> > 我不太认为这是一个很好的想法...
> 
> 个人的建议是讲已经有的提示，从详细设置，改到站点管理页面，让用户可以直接看到问题所在
> 
> 新用户往往不知道需要点站点设置-目录的保存按钮，
> 这会导致例如Wordpress之类的php项目无法上传文件

~~直达链接大法好！~~


其实1Panel的官方文档对于初学者来说其实是够用的状态。如果真的不去看遇到1Panel因不看文档的问题，我认为理当放下使用1Panel的打算，先去学习[提问的智慧](https://github.com/ryanhanwu/How-To-Ask-Questions-The-Smart-Way/blob/main/README-zh_CN.md)才是最好的打算（

如果这条Issue采纳且实装的话，我可提几点建议：
- 不建议直接生硬的加在`创建站点`中，可尝试加在`应用商店`的详细信息中
- 关于本Issue的`Wordpress`的问题，不妨可以在PHP的环境中添加，或在Wordpress安装前弹出关于wp小问题的直达链接

~~不过有些建议说的过于笼统了，可能有点不太懂（~~

**Comment by wanghe-fit2cloud** [comment]
@defeatedperson @ltzXiaoYanMo 
我们会在后续版本考虑上述建议，感谢 👍
